src/components/Signin.tsx

```diff
@@ -11,30 +11,55 @@ import React, { useRef, useState } from 'react';
 import { toast } from 'sonner';
 const Signin = () => {
   const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const [error, setError] = useState({
+    email: '',
+    password: '',
+  });
 
   function togglePasswordVisibility() {
     setIsPasswordVisible((prevState: any) => !prevState);
   }
+
+  function validateForm() {
```

Comment:
maybe you could enforce proper email validation with zod or regex. Since the current implementation only checks if the email and password field is empty or not

Reply:
Just an advice. I could be wrong

Reply:
you can also login using your phone number and password so we should not validate for a valid mail address.